src/environment.js

```diff
@@ -19,7 +19,7 @@ var Environment = Obj.extend({
         // (the full trace from within nunjucks may confuse developers using
         //  the library)
         // defaults to false
-        var opts = this.opts = opts || {};
+        opts = this.opts = opts || {};
```

Comment:
Why are all of these `var`s being dropped?  I don't see them declared anywhere else. I don't like rules that force you declare variables completely separately from where they are used.